Jenkinsfile

```diff
@@ -399,7 +426,7 @@ pipeline {
           if (params.AndroidBindings) {
             iC = dPullOrBuild.dockerPullOrUpdate("android-${params.ABPlatform}-${params.ABBuildType}",
                                                  "${env.GIT_RAW_BASE_URL}/${env.GIT_COMMIT}/docker/android/Dockerfile",
-                                                 "${env.GIT_RAW_BASE_URL}/${env.GIT_PREVIOUS_COMMIT}/docker/android/Dockerfile",
+                                                 "${env.GIT_RAW_BASE_URL}/${previousCommit}/docker/android/Dockerfile",
```

Comment:
Similar change was in 4 places, any ideas how to reduce this?

Reply:
We could introduce a separate variable containing Dockerfile's URL. But this generalization would work unless we are using same URLs for all different types of bindings. In case at some point there is a separate Dockerfile for e.g., Python bindings, we would have to use URL itself and not a variable.